src/bindings/python/tests/test_graph/test_custom_op.py

```diff
@@ -85,11 +86,12 @@ def create_add_model():
 class CustomOpWithAttribute(Op):
     class_type_info = DiscreteTypeInfo("CustomOpWithAttribute", "extension")
 
-    def __init__(self, inputs, attrs):
+    def __init__(self, inputs=None, attrs=None):
         super().__init__(self)
-        self._attrs = attrs
-        self.set_arguments(inputs)
-        self.constructor_validate_and_infer_types()
+        if attrs is not None or inputs is not None:
+            self._attrs = attrs
+            self.set_arguments(inputs)
```

Comment:
**As an alternative if for some reason Sergey's suggestion has technical limitations** (https://github.com/openvinotoolkit/openvino/pull/27664/files#r1879658212) what about a common function? Something like `set_inputs_and_attrs`, which could not only handle the `if attrs is not None or inputs is not None:`, but also call `self.constructor_validate_and_infer_types()`, which is a low level API and could be hidden from the users.

We could migrate from:

```python
def __init__(self, inputs=None, attrs=None):
        super().__init__(self)
        if attrs is not None or inputs is not None:
            self._attrs = attrs
            self.set_arguments(inputs)
            self.constructor_validate_and_infer_types()
```

to:

```python
def __init__(self, inputs=None, attrs=None):
        super().__init__(self)
        self.set_inputs_and_attrs(inputs, attrs)
```